Sources/MediaViewController.swift

```diff
@@ -105,6 +109,19 @@ public class VLCMediaViewController: UICollectionViewController, UISearchResults
     func setupNavigationBar() {
         navigationItem.leftBarButtonItem = UIBarButtonItem(title: NSLocalizedString("SORT", comment: ""), style: .plain, target: self, action: #selector(sort))
     }
+    
+    func setupEmptyBackgroundView() {
+        emptyView = VLCEmptyLibraryView.init()
```

Comment:
you shouldn't have the init() here and should instantiate the View here from the nib